Fixes and general updates (#129)
NCEAS/datateam-training: workflows/edit_eml/edit_attributelists.Rmd

> EML::shiny_attributes()

Reviewer: ```suggestion
atts <- EML::shiny_attributes()
```

```

> Once you are done editing a table in the app, quit the app and the tables will be returned as a list. Alternatively, each table can be to exported to a csv file by clicking the `Download` button.

Reviewer: ```suggestion
Once you are done editing a table in the app, quit the app and the tables will be assigned to the `atts` variable as a list of data frames (one for attributes, factors, and units). Alternatively, each table can be to exported to a csv file by clicking the `Download` button.
```

> Read the table back into your R session and assign it to a variable in your script (e.g. `attributes <- data.frame(...)`).

Reviewer: ```suggestion
If you downloaded the table, read the table back into your R session and assign it to a variable in your script (e.g. `attributes <- data.frame(...)`), or just use the variable that `shiny_attributes` returned.
```
